app/models/book.rb

```diff
@@ -0,0 +1,8 @@
+class Book < ActiveRecord::Base
```

Comment:
## Reek Code Smells Found
[IrresponsibleModule](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md): has no descriptive comment
Lines: [1]
[UncommunicativeVariableName](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md): has the variable name 'a'
Lines: [5]